### PR TITLE
[ADD] estate: Module estate for training purposes T#71276

### PR DESCRIPTION
**Pull request by ralberrto**
- Add module
- Register module as app
- Add Property model to Real Estate App

### CODE REVIEW

**Comment by ralberrto** [comment]
Hi @edy1192 could you checkout this PR, it's ready to be merged. Thank you.

File: estate/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Real Estate ✨",
+    "application": True,
+    "license": "LGPL-3",
+    "author": "Vauxoo",
```
Comment (edy1192) [inline estate/__manifest__.py:5]: Where's the `depends` key? The exercise says that you need to add the base module

image

You can check the documentation here https://www.odoo.com/documentation/16.0/developer/tutorials/getting_started/03_newapp.html#prepare-the-addon-directory in the link of the manifest word.

File: estate/models/estate.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "This model describes the attributes of a property in real estate"
```
Comment (edy1192) [inline estate/models/estate.py:6]: In vauxoo we usually use the model's name in the description. I'll explain why this is later.

```suggestion
    _description = "Estate Property"
```

File: estate/models/estate.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection([
+        ("n", "North"),
+        ("s", "South"),
+        ("e", "East"),
+        ("w", "West")
+    ])
```
Comment (edy1192) [inline estate/models/estate.py:25]: Please use meaningful names.
```suggestion
    garden_orientation = fields.Selection([
        ("north", "North"),
        ("south", "South"),
        ("east", "East"),
        ("west", "West")
    ])
```

**Comment by edy1192** [comment]
@ralberrto please add the commit's description to the PR's description.

**Comment by ralberrto** [comment]
Hey @edy1192 looks like everything has been done. Thank you for your corrections. I'll merge as soon as you tell me to.

File: estate/models/estate_property.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
```
Comment (edy1192) [inline estate/models/estate_property.py:4]: You missed this:

- Class name must match the model name (this is more vauxoo's way).

```suggestion
class EstateProperty(models.Model):
```

**Review by edy1192 — CHANGES_REQUESTED** [review]

**Comment by edy1192** [comment]
> Hey @edy1192 looks like everything has been done. Thank you for your corrections. I'll merge as soon as you tell me to.

Hey @ralberrto You don't have to merge it, I'll be doing it when I consider they are ready.

Regards.

**Comment by ralberrto** [comment]
@edy1192 Thank you! Now I know... Well this time everything appears to be alright, I hope.

**Review by edy1192 — APPROVED** [review]

**Pull request merged**